src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -685,6 +685,14 @@ private void LogInitialization(IHost host, bool isOffline, int attemptCount, int
                 // Reading the string from resources to make sure resource loading code path is warmed up during placeholder as well.
                 logger.InStandByMode(operationId);
             }
+
+            // Log settings
+            var functionWorkerRuntime = _environment.GetEnvironmentVariable(FunctionWorkerRuntime);
+            var functionExtensionVersion = _environment.GetEnvironmentVariable(FunctionsExtensionVersion);
+            var currentDirectory = Directory.GetCurrentDirectory();
```

Comment:
Why do we want to log the current directory? Is it to get a sense of what site extension directory was used to start host? 

I observed that the env variable `ASPNETCORE_IIS_PHYSICAL_PATH` the path to the extension folder (example value: `C:\Program Files (x86)\SiteExtensions\Functions\4.28.3\64bit\`). But this is not available in Linux.

Reply:
You kinda answered the question :-)

Yeah -- we want to know what SiteExtension is running the current process. We surprisingly don't have that information anywhere today unless you join across backend tables.

The goal with this change is to put all the details we need to generate a "host split report" in kusto to see whether we're missing anything when we start to roll this out. Having this all in a single log message makes that a ton easier.